internal/api/v1/services.go

```diff
@@ -0,0 +1,124 @@
+package v1
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/epinio/epinio/internal/cli/clients"
+	"github.com/epinio/epinio/internal/services"
+	"github.com/epinio/epinio/kubernetes"
+	"github.com/julienschmidt/httprouter"
+)
+
+type ServicesController struct {
+}
+
+func (sc ServicesController) ServiceInfo(w http.ResponseWriter, r *http.Request) {
```

Comment:
shouldn't this be the `show` action? Also the route shouldn't need the `/info` part

Reply:
Fixed it hopefully.